From dad2144e61075ca6ae8b52a53aa52946fc273f53 Mon Sep 17 00:00:00 2001
From: "L. E. Segovia" <amy@centricular.com>
Date: Mon, 1 Apr 2024 23:55:39 -0300
Subject: [PATCH 10/12] glib/gthread-posix: Fix name of the futex_time64(2)
 test in the Meson log

(cherry picked from commit ffa639f0b7aae99cffd657bf306358dd623ca9d0)
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index be468c3f3..4c2c150a7 100644
--- a/meson.build
+++ b/meson.build
@@ -900,7 +900,7 @@ if cc.compiles('''#include <linux/futex.h>
                int main (int argc, char ** argv) {
                  syscall (__NR_futex_time64, NULL, FUTEX_WAKE, FUTEX_WAIT);
                  return 0;
-               }''', name : 'futex(2) system call')
+               }''', name : 'futex_time64(2) system call')
   glib_conf.set('HAVE_FUTEX_TIME64', 1)
 endif
 
-- 
2.44.0.windows.1

